integration-tests/actions/vrf/common/logging_helpers.go

```diff
@@ -0,0 +1,129 @@
+package common
+
+import (
+	"fmt"
+	"math/big"
+
+	"github.com/rs/zerolog"
+
+	commonassets "github.com/smartcontractkit/chainlink-common/pkg/assets"
+	"github.com/smartcontractkit/chainlink/integration-tests/contracts"
+	"github.com/smartcontractkit/chainlink/v2/core/chains/evm/assets"
+	"github.com/smartcontractkit/chainlink/v2/core/gethwrappers/generated/vrf_owner"
+	"github.com/smartcontractkit/chainlink/v2/core/gethwrappers/generated/vrfv2plus_wrapper_load_test_consumer"
+)
+
+func LogSubDetails(
+	l zerolog.Logger,
+	subscription contracts.Subscription,
+	subID string,
+	coordinator contracts.Coordinator,
+) {
+	log := l.Info().
+		Str("Coordinator", coordinator.Address()).
+		Str("Link Balance", (*commonassets.Link)(subscription.Balance).Link()).
+		Str("Subscription ID", subID).
+		Str("Subscription Owner", subscription.SubOwner.String()).
+		Interface("Subscription Consumers", subscription.Consumers)
+	if subscription.NativeBalance != nil {
```

Comment:
LINK balance can also be zero?

Reply:
it shouldnt. this is only when V2 is used and there is no Native